### ISSUE TITLE
winbuilder error

### ISSUE THREAD

**Issue by ShanaScogin** [ShanaScogin/BayesPostEst#78]
There seem to be a new winbuilding error (see below). I'm not sure if this is related to the email from CRAN in #77 yet. Will update soon.

```
make cmd is
  make -f "D:/RCompile/recent/R/etc/x64/Makeconf" -f "D:/RCompile/recent/R/etc/x64/Makevars.site" -f "D:/RCompile/recent/R/share/make/winshlib.mk" CXX='$(CXX14) $(CXX14STD)' CXXFLAGS='$(CXX14FLAGS)' CXXPICFLAGS='$(CXX14PICFLAGS)' SHLIB_LDFLAGS='$(SHLIB_CXX14LDFLAGS)' SHLIB_LD='$(SHLIB_CXX14LD)' SHLIB="file3f7c241da26630.dll" WIN=64 TCLBIN=64 OBJECTS="file3f7c241da26630.o"

make would use
make[1]: Entering directory '/d/temp/RtmpOc4Ifn'
if test "zfile3f7c241da26630.o" != "z"; then \
  if test -e "file3f7c241da26630-win.def"; then \
    echo d:/Compiler/rtools40/mingw64/bin/g++ -shared -s -static-libgcc -o file3f7c241da26630.dll file3f7c241da26630-win.def file3f7c241da26630.o -L"d:/Compiler/gcc-4.9.3/local330/lib/x64" -L"d:/Compiler/gcc-4.9.3/local330/lib" -L"D:/RCompile/recent/R/bin/x64" -lR ; \
    d:/Compiler/rtools40/mingw64/bin/g++ -shared -s -static-libgcc -o file3f7c241da26630.dll file3f7c241da26630-win.def file3f7c241da26630.o -L"d:/Compiler/gcc-4.9.3/local330/lib/x64" -L"d:/Compiler/gcc-4.9.3/local330/lib" -L"D:/RCompile/recent/R/bin/x64" -lR ; \
  else \
    echo EXPORTS > tmp.def; \
    d:/Compiler/rtools40/mingw64/bin/nm file3f7c241da26630.o | sed -n 's/^.* [BCDRT] / /p' | sed -e '/[.]refptr[.]/d' -e '/[.]weak[.]/d' | sed 's/[^ ][^ ]*/"&"/g' >> tmp.def; \
    echo d:/Compiler/rtools40/mingw64/bin/g++ -shared -s -static-libgcc -o file3f7c241da26630.dll tmp.def file3f7c241da26630.o -L"d:/Compiler/gcc-4.9.3/local330/lib/x64" -L"d:/Compiler/gcc-4.9.3/local330/lib" -L"D:/RCompile/recent/R/bin/x64" -lR ; \
    d:/Compiler/rtools40/mingw64/bin/g++ -shared -s -static-libgcc -o file3f7c241da26630.dll tmp.def file3f7c241da26630.o -L"d:/Compiler/gcc-4.9.3/local330/lib/x64" -L"d:/Compiler/gcc-4.9.3/local330/lib" -L"D:/RCompile/recent/R/bin/x64" -lR ; \
    rm -f tmp.def; \
  fi \
fi
make[1]: Leaving directory '/d/temp/RtmpOc4Ifn'
Quitting from lines 205-211 (getting_started.Rmd)
Quitting from lines 205-211 (getting_started.Rmd)
Error: processing vignette 'getting_started.Rmd' failed with diagnostics:
invalid connection
--- failed re-building 'getting_started.Rmd'

SUMMARY: processing the following file failed:
  'getting_started.Rmd'

Error: Vignette re-building failed.
Execution halted
```

(Also thanks to Naidan for pointing it out!)

**Comment by ShanaScogin** [comment]
Quick update: lines 205-211 in the vignette is `stan()` from rstan

```
fit.stan <- stan(file = "mod.stan",  
           data = dl,         
           pars = c("b"),     
           chains = 4,        
           iter = 2000,       
           seed = 123)    
```

**Comment by ShanaScogin** [comment]
~The warning seems to have been fixed with 99c6e7b57e00a8c4005f9741bd210fae63f74f0c, though now a NOTE exists. Since it does seem to be related to #77, so I'm going to go ahead and close this.~

Leaving this closed, but the issue seems to be with rstan. I have commented out the rstan code in the vignette with 9140772504402f448511ccff0186661bf1b6188c until it clears up.
